tuxemon/item/effects/evolve.py

```diff
@@ -42,10 +42,9 @@ class EvolveEffect(ItemEffect):
     monster_evolve: str
 
     def apply(self, target: Monster) -> EvolveEffectResult:
-        monster_evolve = self.monster_evolve
-
-        if any(d["monster_slug"] == monster_evolve for d in target.evolutions):
-            self.user.evolve_monster(target, monster_evolve)
-            return {"success": True}
-        else:
-            return {"success": False}
+        for evolution in target.evolutions:
+            if evolution.monster_slug == self.monster_evolve:
+                self.user.evolve_monster(target, self.monster_evolve)
+                return {"success": True}
+            else:
+                return {"success": False}
```

Comment:
This will only ever check the first evolution

Reply:
First and only one since it's an object. I'm considering to remove the "else", because at the end you select the monster in the previous passage, so there is no alternative.

Reply:
Yeah, the only exception is botbot

Reply:
Luckily for us booster_tech is the only way botbot evolves